src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyRepositoryTests.cs

```diff
@@ -4,220 +4,37 @@
-            };
-
-            var methodology2 = new Methodology
+            await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContextId))
             {
-                Title = "Test methodology 2",
-            };
-            var publication2 = new Publication
-            {
-                Title = "Test pub 2",
-                Methodology = methodology2
-            };
-            var release2 = new Release
-            {
-                Publication = publication2
-            };
-            var userReleaseRole2 = new UserReleaseRole
-            {
-                UserId = userId,
-                Release = release2,
-                Role = ReleaseRole.Contributor,
-            };
-
-            // Not associated with userId
-            var methodology3 = new Methodology
-            {
-                Title = "Ignored methodology 3",
-            };
-            var publication3 = new Publication
-            {
-                Title = "Test pub 3",
-                Methodology = methodology3
-            };
-            var release3 = new Release
-            {
-                Publication = publication3
-            };
-            var userReleaseRole3 = new UserReleaseRole
-            {
-                UserId = Guid.NewGuid(),
-                Release = release3,
-                Role = ReleaseRole.Lead,
-            };
+                var service = BuildMethodologyRepository(contentDbContext);
 
-            var methodology4 = new Methodology
-            {
-                Title = "Ignored methodology 4",
-            };
-
-            var contentDbContextId = Guid.NewGuid().ToString();
-            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
-            {
-                await contentDbContext.AddRangeAsync(
-                    userReleaseRole1,
-                    userReleaseRole2,
-                    userReleaseRole3,
-                    methodology4);
-                await contentDbContext.SaveChangesAsync();
-            }
-
-            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
-            {
-                var methodologyRepository = BuildMethodologyRepository(contentDbContext);
-                var methodologies = await methodologyRepository.GetMethodologiesForUser(userId);
+                var result = await service.GetLatestMethodologiesByRelease(release.Id);
 
-                Assert.Equal(2, methodologies.Count);
-                Assert.Contains(methodologies, m => m.Id == methodology1.Id);
-                Assert.Contains(methodologies, m => m.Id == methodology2.Id);
-                Assert.DoesNotContain(methodologies, m => m.Id == methodology3.Id);
-                Assert.DoesNotContain(methodologies, m => m.Id == methodology4.Id);
+                // TODO SOW4 EES-2379 Test getting latest methodologies for a Release for approval checklist
```

Comment:
I'd be tempted to leave the correct assertion in this test (and any other test that has had to change temporarily) as it was (if it was correct still) and move this TODO to a [Skip] annotation on the test method.

Saying that it's not a biggie.